rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/terminal/ITerminalIDEServer.java

```diff
@@ -304,7 +304,7 @@ public static String value2string(IValue value) {
             IValueOutputStream out = new IValueOutputStream(stream, IRascalValueFactory.getInstance());
         ) {
             out.write(value);
-            out.close();
+            // out.close(); removed this because of a compiler warning. seems superfluous anyway.
```

Comment:
I disagree, we do this so that the stream flushes. maybe we should move the return statement outside the try, but in this way there could be unflushed stuff.

Reply:
ok, that is true and if there is no buffering now there might be in the future.